app/Resources/views/pagelayout.html.twig

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en-US"
+      class="no-js">
```

Comment:
please, do not break lines in the way like this. It makes everything less readable 😢